content/en/blog/2025/stability-proposal-announcement.md

```diff
@@ -0,0 +1,180 @@
+
+<!-- If your post doesn't have secondary authors, then delete the following paragraph: -->
+
+With contributions from, and on behalf of, the OpenTelemetry Governance
+Committee.
+
+OpenTelemetry is a large, complex, and highly involved project. A quick glance
+at the
+[specification compliance matrix](https://github.com/open-telemetry/opentelemetry-specification/blob/main/spec-compliance-matrix.md)
+should be enough to tell you that supporting tracing, metrics, logs, and
+profiles across a dozen languages – not to mention the complexity that comes
+from different language runtimes and execution environments – means that there’s
+a _lot_ to take in. One thing we’ve heard from you over the past months is that
+this complexity makes it difficult to get started, or get going, with
+OpenTelemetry as an end user. It also makes life difficult for our maintainers
+and contributors. How should they prioritize feature development versus spec
+changes versus instrumentation stability?
+
+With this in mind, the OpenTelemetry project would like to announce some
+proposed changes that we’d like to institute, with the goal of normalizing our
+language around component stability and component offerings.
+
+We believe that these changes need community involvement and discussion to be a
+success, so we’re taking this opportunity to announce our intention and open an
+[issue for discussion](https://github.com/open-telemetry/community/issues/3086)
+in order to get feedback from users, maintainers, and contributors. We do not
+anticipate that these changes will be completed overnight, and want to assure
+everyone that we will continue to prioritize our existing commitments to users
+and maintainers even as we consider necessary changes for the overall wellbeing
+and maturity of the project.
+
+## Improving defaults for stable components
+
+The first proposal has to do with how SIGs and the project communicate stability
+of components – such as instrumentation libraries, Collector receivers, or tools
+like weaver. We will clearly state on the project website which components are
+stable. We may list un-stable components as well, but the main goal is to
+provide end-users with a single source of truth about what they can expect to be
+stable. Stability requirements will also be streamlined and clarified for
+contributors, maintainers, and end-users. We plan to expand these stability
+requirements to include documentation, examples, implementation scenarios,
+performance benchmarks, and other useful information.
+
+In addition, we are proposing a change to the defaults of the SDK and
+instrumentation agents. By default, stable components should not automatically
+import, include, or enable unstable components. For example, if you install a
+stable distribution of OpenTelemetry Java Instrumentation Agent, the default
+receivers, processors, samplers, exporters, and libraries should not include
+alpha or experimental instrumentations (which is the current behavior).
```

Comment:
Not only is this the current behavior for the Java Instrumentation Agent, this is also the a[pproach that we announced that we would be following for Android](https://opentelemetry.io/blog/2025/android-road-to-stable/#looking-ahead). This change injects chaos into that plan, so we will need to pause this plan until this is figured out.

Reply:
well, if you're planning on doing what the java agent does, then I think you'd be fine...?

Reply:
oh, sorry, @austinlparker I think I got our wires crossed when I answered your question the other day

the Java agent current behavior _does_ include unstable things by default